pkg/kapp/logs/pod_container_log.go

```diff
@@ -140,6 +163,16 @@ func (l PodContainerLog) readyToGetLogs() bool {
 	if err != nil {
 		return false
 	}
+	containerStatuses := pod.Status.ContainerStatuses
+	for _, containerStatus := range containerStatuses {
+		//We should focus only on the status of this particular container.
+		if l.container != containerStatus.Name {
+			continue
+		}
```

Comment:
Is this an old comment that you forgot to remove? As it says we should focus only on this particular container but we are just doing a continue in the if condition.

Reply:
I believe it can be better phrased as "Filter out statuses from containers other than the one being tailed"
I believe a pod might have statuses from multiple containers and this means that we are only looking at ones from the container being observed

Reply:
Yeah, I did think that @rohitagg2020 started with if `l.container == containerStatus.Name` and then changed it to `if l.container != containerStatus.Name`, but forgot to update the comment later on. 

> "Filter out statuses from containers other than the one being tailed"

Makes sense. Although I would recommend to remove the comment as the if condition seems self explanatory to me.